### PR TITLE
feat(sidebar_footer_image): add sidebar footer image config

### PR DESCRIPTION
**Pull request by simonsmh**
**Contributing rules**

- Fork the repo and create your branch from `canary`. Then be sure to put the `canary` branch as the target for your pull request.
- Please be sure to follow the [contributing guidelines](https://github.com/viosey/hexo-theme-material/blob/master/CONTRIBUTING.md), especially for commit message
- Remove the `Contributing rules` part from this description
- Fill out the other parts from this description

**What kind of change does this PR introduce?** (check one with "x")

- [ ] Bug fix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?** (check one with "x")

- [x] Yes
- [ ] No

____

**Description**

Add footer image config.
Now users are able to add their own footer image.
#140 #141 
image
to review
____

**Verification steps**

### CODE REVIEW

File: layout/_partial/sidebar-footer_image.ejs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<% if(theme.img.footer_image) { %>
+    <div id="footer-image">
```
Comment (pidupuis) [inline layout/_partial/sidebar-footer_image.ejs]: - I suggest you to put the `div` inside the `for` loop
- Please change `#scheme-Paradox #upyun-logo` by `#scheme-Paradox #footer-image` in `style.css`
- Please test with multiple images to adapt the style

Comment (simonsmh) [inline layout/_partial/sidebar-footer_image.ejs]: Hmmm...
In one div block it's horizontal so multiple pictures show well.
In multiple div block it's vertical and only show one picture. The next one have to be scroll to show.
I will try to fix it although I have no idea what is leading to such a strange behavior.

Comment (pidupuis) [inline layout/_partial/sidebar-footer_image.ejs]: Maybe try to change the `div` for a `span` and put them inside the loop

**Comment by simonsmh** [comment]
Good to go?
It seems working properly.

**Pull request merged**